samcli/commands/local/cli_common/invoke_context.py

```diff
@@ -404,6 +404,7 @@ def _get_stacks(self) -> List[Stack]:
                 self._template_file,
                 parameter_overrides=self._parameter_overrides,
                 global_parameter_overrides=self._global_parameter_overrides,
+                root_template_path=os.path.dirname(os.path.abspath(self._template_file)),
```

Comment:
Should we add a comment on why a root template path is required?

Reply:
Sure! Since this pr is a little outdated and the branch is deleted. I will recreated a PR for this change.